### PR TITLE
Add paritydb section to Forest config

### LINKED ISSUE #2301: Add support for ParityDb configuration
**Issue summary**

ParityDb doesn't have a lot of knobs but there are some that we would like to expose to the user in our config file. Ie:

```
[parity_db]
stats = false
compression = "lz4" # other possible values are "snappy" and "none"
```

**Task summary**
- [ ] Add similar `parity_db` section than above to forest `Config` struct
- [ ] Change `stats` to `false` by default (to avoid any potential overhead)

**Acceptance Criteria**
- [ ] We can enable statistics manually
- [ ] We can dump the default config file with `config dump`, tweak `parity_db` section and load back the .toml
- [ ] Changing compression works

**Other information and links**

https://docs.rs/parity-db/latest/parity_db/struct.Options.html

### PR DESCRIPTION
**Pull request by elmattic**
**Summary of changes**
Changes introduced in this pull request:
- Expose `stats` and `compression` settings
- Disable `stats` by default

**Reference issue to close (if applicable)**

Closes #2301

**Other information and links**

### CODE REVIEW

File: node/db/src/parity_db.rs [hunk 1]
```diff
@@ -154,3 +169,27 @@ impl BitswapStore for ParityDb {
...
 impl DBStatistics for ParityDb {}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use parity_db::CompressionType;
+
+    #[test]
+    fn compression_type_from_str_test() {
+        let test_cases = vec![
```
Comment (LesnyRumcajs) [inline node/db/src/parity_db.rs:180]: static array should do fine
```suggestion
        let test_cases = [
```

**Review by LesnyRumcajs — APPROVED** [review]

File: node/db/src/parity_db.rs (hunk 1 above)
Comment (elmattic) [reply node/db/src/parity_db.rs:180]: alright

File: forest/shared/src/cli/config.rs [hunk 2]
```diff
@@ -149,6 +149,8 @@ pub struct Config {
     pub client: Client,
     #[cfg(feature = "rocksdb")]
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
+    #[cfg(feature = "paritydb")]
+    pub parity_db: forest_db::parity_db::ParityDbConfig,
```
Comment (lemmih) [inline forest/shared/src/cli/config.rs:153]: Let's make it part of the configuration even if `paritydb` is disabled. That way, the configuration won't be invalid if you switch between databases.

Comment (elmattic) [reply forest/shared/src/cli/config.rs:153]: How exactly?

Comment (lemmih) [reply forest/shared/src/cli/config.rs:164]: By removing the `#[cfg(feature = "paritydb")]` line above it.

**Review by lemmih — APPROVED** [review]

**Pull request merged**